common/src/main/java/com/wynntils/wynn/model/CompassModel.java

```diff
@@ -48,9 +48,6 @@ public static Optional<WaypointPoi> getCompassWaypoint() {
             WaypointPoi waypointPoi = new WaypointPoi(() -> {
                 Location location = locationSupplier.get();
 
-                // Make sure to always render on top
-                location.set(location.x, Double.MAX_VALUE, location.z);
-
```

Comment:
This was the problem, we did not even need this as rendering code already handled it. @Incompleteusern

Reply:
Thanks!

Reply:
I've been saying for a long time that this was fishy code. ;-)